src/torchjd/autojac/mtl_backward.py

```diff
@@ -0,0 +1,180 @@
+from typing import Iterable, Sequence
+
+from torch import Tensor
+
+from torchjd.aggregation import Aggregator
+
+from ._transform import (
+    EmptyTensorDict,
+    Grad,
+    Gradients,
+    Init,
+    Jac,
+    Stack,
+    Store,
+    Subset,
+    Transform,
+    make_aggregation,
+)
+from ._transform.strategy import UnifyingStrategy
+from ._utils import (
+    _as_tensor_list,
+    _check_optional_positive_chunk_size,
+    _check_retain_graph_compatible_with_chunk_size,
+)
+
+
+def mtl_backward(
+    features: Sequence[Tensor] | Tensor,
+    losses: Sequence[Tensor],
+    shared_params: Iterable[Tensor],
+    tasks_params: Sequence[Iterable[Tensor]],
+    A: Aggregator,
+    retain_graph: bool = False,
+    parallel_chunk_size: int | None = None,
+) -> None:
+    r"""
+    In the context of Multi-Task Learning (MTL), we often have a shared model to extract features,
+    followed by several task-specific heads. A loss can then be computed for each task.
+
+    This function computes the gradient of each element of ``losses`` with respect to the
```

Comment:
For each task, this function computes the gradient of the loss with respect to the task-specific parameters and stores it in their .grad field.
Then, it computes the Jacobian of all losses with respect to the shared parameters. It aggregates it using A and stores the result in their .grad field.